### ISSUE TITLE
[BUG] PyPlot matplotlib 3.3 deprecations

### ISSUE THREAD

**Issue by isentropic** [JuliaPlots/Plots.jl#2874]
Only relevant to pyplot with most recent matplotlib
```
plot(1:10)
sys:1: MatplotlibDeprecationWarning: Passing the fontdict parameter of _set_ticklabels() positionally is deprecated since Matplotlib 3.3; the parameter will become keyword-only two minor releases later.
sys:1: UserWarning: FixedFormatter should only be used together with FixedLocator
sys:1: MatplotlibDeprecationWarning: Case-insensitive properties were deprecated in 3.3 and support will be removed two minor releases later
```

**Comment by John-Boik** [comment]
I am getting the same warnings.  Until the issue is fixed, is there any way to suppress the warnings?

**Comment by lewisl** [comment]
Any word on this?

**Comment by isentropic** [comment]
@John-Boik @lewisl  please report on this PR, I think i got this resolved. I would really appreciate if someone can test this with older pyplot so that it does not break back compatibility

**Comment by lewisl** [comment]
How do I install with package manager?

**Comment by lewisl** [comment]
So, I couldn’t test with the mentioned PR. I just updated and it appears that this closed.

Just ran it and got:

```
sys:1: MatplotlibDeprecationWarning: Passing the fontdict parameter of _set_ticklabels() positionally is deprecated since Matplotlib 3.3; the parameter will become keyword-only two minor releases later.
sys:1: UserWarning: FixedFormatter should only be used together with FixedLocator
```

**Comment by isentropic** [comment]
@lewisl 
you'd need to add this `https://github.com/isentropic/Plots.jl/tree/pyplot-deprecations` in your package manager and see if it works. There was no new release of Plots yet, so this fix is available by `add Plots`, but I think you could getaway with `add Plots#master`

**Comment by lewisl** [comment]
Just tested.

No deprecation messages from matplotlib. Seems to generate first chart much faster too.

Awesome!

Hope this will be released.

**Comment by TheC0d3Mast3r** [comment]
Hi guys,
Can someone exactly say what I need to do to suppress the annoying warnings?

**Comment by daschw** [comment]
> Can someone exactly say what I need to do to suppress the annoying warnings?

This should be fixed in Plots 1.6.9.
